### PR TITLE
Pla 1072 add more colours to study move notation

### PR DESCRIPTION
**Pull request by JamesHeppell**

### CODE REVIEW

File: ui/tree/css/_tree.scss [hunk 1]
```diff
@@ -65,7 +95,7 @@ $c-blunder: #df5353;
     font-style: italic;
   }
 
-  move:not(.empty):hover {
+  move:not(.empty):not(.blunder):not(.mistake):not(.inaccuracy):not(.good):not(.brilliant):not(.interesting):hover {
```
Comment (vincentfrochot) [inline ui/tree/css/_tree.scss:98]: is it possible to have the list inside the :not selector ?

```suggestion
  move:not(.empty, .blunder, .mistake, .inaccuracy, .good, .brilliant, .interesting):hover {
```

Comment (vincentfrochot) [reply ui/tree/css/_tree.scss:98]: Apparently not ?
image

Comment (vincentfrochot) [reply ui/tree/css/_tree.scss]: It looks like major browsers are supporting it with a version released ~ end of 2020.
If you want to ensure people using Opera Android or old browsers, you can revert.

Comment (JamesHeppell) [reply ui/tree/css/_tree.scss]: okay, its a minor on hover change, so will just leave as is.

**Pull request merged**